package.json

```diff
@@ -44,7 +44,9 @@
     "autosize": "3.0.21",
     "base64-image-loader": "1.2.0",
     "flatpickr": "3.0.6",
-    "prop-types": "15.5.10"
+    "prop-types": "15.5.10",
+    "react-redux": "5.0.6",
+    "redux-form": "7.0.3"
```

Comment:
I actually don't think you need `react-redux` and `redux-form`. I have an [open PR](https://github.com/meetup/meetup-web-components/pull/267/files#diff-ccd4c1d6611298c48e393c9ad13d8148) at the moment for a radio button and radio button group that has a redux-form wrapper that might be a good point of reference for how to accomplish this without these dependencies. Basically, I'm just mocking the props that redux-form sends through to the component instead of actually hooking everything up to redux-form. That _should_ make your tests much more simple and reliable too.